tests/testsuite/messages.rs

```diff
@@ -48,6 +46,20 @@ pub fn raw_rustc_output(project: &Project, path: &str, extra: &[&str]) -> String
     result
 }
 
+fn redact_rustc_message(msg: &str) -> String {
```

Comment:
I think a core part of the problem is that we don't redact expected values.  Maybe we should revisit that?

Reply:
Indeed! I think providing a function to normalize/redact the expected value manually or a toggle to automatically normalize expected value would be more flexible and less troublesome than always normalizing, as there might be cases where we don't want this behavior.

Reply:
#353

Reply:
Ah, I think you're referring to https://github.com/assert-rs/snapbox/pull/353?